tools/check_format.py

```diff
@@ -10,6 +10,9 @@
                      "./bazel-")
 SUFFIXES = (".cc", ".h", "BUILD")
 
+# Files in these paths can make reference to protobuf stuff directly
+GOOGLE_PROTOBUF_WHITELIST = ('./ci/prebuilt', './source/common/protobuf')
```

Comment:
small ask: This is going to cause issues for us in our import. Would it be possible to do this as a regex somehow where it will match an arbitrary prefix? Or allow the prefix to be passed as a new CLI option?